pyccl/tests/test_swig_interface.py

```diff
@@ -102,7 +102,7 @@ def test_swig_cls():
         ccllib.angular_cl_vec,
         COSMO,
         None, None, None,
-        1, 0,
+        1, 0, 501,
```

Comment:
Can we use the enum? This magic number is confusing.

Reply:
done (not that it matters, since all the inputs here are crap)